fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1003,8 +1003,18 @@ public Void visitFunctionCall(FunctionCallExpr node, Scope scope) {
 
             // throw exception direct
             checkFunction(fnName, node, argumentTypes);
-
-            if (fnName.equals(FunctionSet.COUNT) && node.getParams().isDistinct()) {
+            if (fnName.equalsIgnoreCase("typeof") && argumentTypes.length == 1) {
```

Comment:
You should also add check and test case for `select typeof(cast(1 as tinyint),  cast(1 as int);`

Reply:
I have added a test at test/sql/test_function/test_typeof. Do you mean to test multiple typeof functions in a select?

Reply:
I mean that give the typeof function multi arguments, which is invalid.

Reply:
got it